ft(feature):create a political party

Hoxtygen opened the pull request:
#### What does this PR do?
Add a new route to the endpoint.This new route allows the creation of
new political office.

#### Description of Task to be completed?
Have the following endpoint working
`POST /api/v1/parties/`

#### How should this be manually tested?
  clone the repo and cd into it then on the terminal do `npm run dev`
Using postman, test the endpoint with the following header information
- key: Content-Type value: application/json


#### What are the relevant pivotal tracker stories?
[Delivers [#163393386](https://www.pivotaltracker.com/story/show/163393386)]

#### Screenshots (if appropriate)


![createparty](https://user-images.githubusercontent.com/19408669/51743203-3b191f80-209c-11e9-8d8c-bbaf0ec35e8c.PNG)

Hoxtygen committed:
ft(feature):create a political party

Add a new route to the endpoint.This new route allows the creation of
new political office.

[Delivers #163393386]

Orlayhemmy commented on server/controllers/partiesController.js:
```diff
@@ -43,6 +43,31 @@ class partyController {
+      logoUrl,
+    };
+    Parties.push(newParty);
+    console.log();
```
Remove the log

Hoxtygen replied:
Okay. That one got away.

Orlayhemmy commented on server/controllers/partiesController.js:
```diff
@@ -43,6 +43,31 @@ class partyController {
     });
   }
+
+  static addNew(req, res) {
```
Add new what? Give a better variable naming

Hoxtygen replied:
Copy that.

Hoxtygen committed:
refactor:rename route and remove console message

Give the route a more semantic meaning that reflects what the route
actually does and remove the console.log() line for a more cleaner code.

[Delivers #163393386]
